feature/policy_forwarding/GUE/Outer_IPv4/decapsulation/otg_tests/Inner_MPLS/README.md

```diff
@@ -0,0 +1,104 @@
+# PF-1.5: Interface based MPLSoGUE Decapsulation to IPv4 tunnel
+
+## Summary
+
+This is to test the the functionality of policy-based forwarding (PF) to decapsulate Generic UDP Encapsulation (GUE) traffic. These tests verify the use case of MPLSoGUE to IPv4 GUE tuennel. The tests are meant for `Tunnel Interface` or `Policy Based` implementation of IPv4 GUE tunnel. The tests validate that the DUT performs the following action.
```

Comment:
Per comments elsewhere, I'm not sure that I understand why we have these variants -- this seems like a deviation for the same functionality.